crates/solver/src/liquidity/zeroex.rs

```diff
@@ -48,7 +48,7 @@ impl ZeroExLiquidity {
     ) -> Self {
         let gpv2_address = gpv2.address();
         let allowance_manager = AllowanceManager::new(web3, gpv2_address);
-        let orderbook_cache = Arc::new(RwLock::new(vec![]));
+        let orderbook_cache = Arc::new(ArcSwap::from_pointee(Vec::new()));
```

Comment:
Using `Default::default()` somewhere here should work.
```suggestion
        let orderbook_cache = Default::default();
```